spec/player_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+RSpec.describe Player do
+  describe '#initialize' do
+    it 'exists' do
+      player = Player.new
+
+      expect(player).to be_a(Player)
+      expect(player.symbol).to be_a(String)
+      expect(player.input).to eq(nil)
+    end
+  end
```

Comment:
Also, what are your thoughts on a second/edge case(?) test for #initialize so that "first test is not a coincidence"?? I've seen the instructors demo it in class a few times so I also started adding it, but do we want to consider it for our tests? 

Otherwise, these are the three expects I would have done, so looks good to me! :)

Reply:
I would like to test for the second edge case!

Reply:
Okay! I will add the second test for Computer class as well!